Changelog.md

```diff
@@ -4,6 +4,12 @@
 
 Nothing here yet.
 
+## v4.3.1 (2019-03-16)
```

Comment:
please do not create this version section here. You'd better rebase from the latest master branch, and add your changelog entry under the ``master (unreleased)`` section.